docs/ref-info/glossary-uid.md

```diff
@@ -334,6 +346,11 @@ import Link from '@docusaurus/Link';
 <dd>For details, see <a href="https://blog.google/products/admanager/new-ways-for-publishers-to-manage-first-party-data/">Share secure signals with your trusted partners</a> (second section) and <a href="https://support.google.com/admanager/answer/10488752?hl=en">Share secure signals with bidders</a>, both from Google.</dd>
 <dd>For details about UID2 support of the Google secure signals feature, see <a href="../guides/google-ss-integration">Google Ad Manager Secure Signals Integration Guide</a>.</dd>
 
+<dt><MdxJumpAnchor id="gl-server-side">Server-side integration</MdxJumpAnchor></dt>
+<dd>UID2 integrations can be entirely on the <a href="#gl-client-side">client side</a>, entirely on the server side, or partially on the client side and partially on the server side (<a href="#gl-client-server">client-server</a>).</dd>
+<dd>In a server-side integration, all activities relating to UID2, such as generating and refreshing UID2 tokens, are performed on the server.</dd>
+<dd>For example, in a server-side integration, advertisers generate raw UID2s on the server side to be delivered for audience targeting, and publishers generate UID2 tokens on the server side for bidstream use.</dd>
```

Comment:
maybe link to docs/guides/custom-publisher-integration ?

Reply:
Done, thx.